AngularJS/reportList.html

```diff
@@ -0,0 +1,77 @@
+<div class="table-responsive">
+  <button class='btn btn-primary btn-xs' ng-click="$ctrl.form=!$ctrl.form;$ctrl.editForm = true;">Add
```

Comment:
do not write more than one JS statement in template. create a function for this.